doc/api/inspector.md

```diff
@@ -383,6 +383,69 @@ session.post('HeapProfiler.takeHeapSnapshot', null, (err, r) => {
 });
 ```
 
+### Class: `inspector.RawSession`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Extends: {EventEmitter}
+
+Both `RawSession` and `Session` are used to communicate with the V8 Inspector.
+But they're a little different.
+
+When the user call `Session.post` to dispatch a message to V8 inspector,
+`Session.post` will construct a request that conforms to the format of the
+V8 inspector protocol. When `Session` receives a message from the V8 inspector.
+It will parse the data and execute callback passed in by the user or trigger
+the corresponding event.
```

Comment:
```suggestion
When using `Session.post()` to dispatch a message to V8 inspector,
`Session.post()` will construct a request that conforms to the format of the
V8 inspector protocol. When `Session` receives a message from the V8 inspector.
It will parse the data and execute a callback passed in by the user or trigger
the corresponding event.
```

Reply:
Done, thanks !